ranker/shared/ranker_obj.py

```diff
@@ -149,6 +154,7 @@ def score(self, answer, jaccard_like=False):
                 continue
 
             score = np.exp(-kirchhoff(laplacian, probes))
+            print(laplacian, probes, score)
```

Comment:
Can we take this out?

Reply:
Ha! No way!! This one is a load bearing print statement. This is here to slow down runtime so as to avoid a race condition with ....

Jk. Of course. My bad.